fs_gcsfs/__init__.py

```diff
@@ -1,4 +1,10 @@
 # flake8: noqa
+from pkg_resources import get_distribution, DistributionNotFound
 
 from ._gcsfs import GCSFS
-from ._version import __version__
+
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
+    # package is not installed
+    pass
```

Comment:
I am slightly unsure what we mean by "# package is not installed", i.e. is it reasonable / safe to ignore the `DistributionNotFound` and live with `__version__` not being defined?

Reply:
This is snippet is taken from the [setuptools-scm docs](https://pypi.org/project/setuptools-scm/).

Exposing `__version__` in the init is nothing one has to do, it's just a convention that some libraries follow. If our lib is installed regulary as a pip or conda package, `__version__` will be available. If someone decides to e.g. download this repo as a zip (without scm info) it will at least work an not throw an exception on import. Not exposing the version is the most reasonable thing to do in this case imho.

Reply:
Your description above would be a perfect comment so we don't ask ourselves the same question at some point in the future.